src/helpers/utilities.ts

```diff
@@ -451,6 +451,43 @@ export const omitFlatten = <T extends object, K extends keyof T>(
   return newObj;
 };
 
+/**
+ * @desc Converts the first character of string to upper case and the remaining to lower case
+ */
+export const capitalize = (string?: string | undefined) => {
+  if (!string) return '';
+  return string
+    ? string.charAt(0).toUpperCase() + string.slice(1).toLowerCase()
+    : '';
+};
+
+/**
+ * @desc The passed value is converted to a boolean value, if necessary.
+ * If the value is omitted or is 0, -0, null, false, NaN, undefined, or the empty string (""),
+ * the object has an initial value of false. All other values,
+ * including any object, an empty array ([]), or the string "false",
+ * create an object with an initial value of true.
+ */
+export const toBoolean = <T>(argument: T | undefined): argument is T => {
+  return Boolean(argument);
+};
```

Comment:
This function is problematic because of this this reason 
![image](https://user-images.githubusercontent.com/25709300/179476712-6c65c59f-0188-41c6-b11f-d7db3ca47ee6.png)

Reply:
As discussed in the call, this is a thing for later.

Reply:
Okay, so for now, I use `Boolean` instead of this implementation with `isBoolean`